src/rdgz.h

```diff
@@ -41,5 +41,5 @@
  */
 void *rd_gz_decompress (const void *compressed, int compressed_len,
 			uint64_t *decompressed_lenp);
-
+#endif
```

Comment:
What about this line ? 
Need to be `endif /* WITH_ZLIB */` ?

Reply:
yes of course

Reply:
Done!